packages/components/src/notice/index.js

```diff
@@ -84,6 +88,16 @@ function Notice( {
 		children = <RawHTML>{ children }</RawHTML>;
 	}
 
+	onDismiss = onDismiss || noop;
+
+	const dismissMe = ( event ) => {
```

Comment:
I know this is the same name with the other PR, but let's make it a bit more self explaining with something like `onDismissNotice` or something like that.